packages/clerk-js/src/ui/SignIn/SignInFactorTwoAlternativeMethods.tsx

```diff
@@ -80,6 +80,8 @@ export function getButtonLabel(factor: SignInFactor): string {
       return `Send code to ${formatSafeIdentifier(factor.safeIdentifier)}`;
     case 'totp':
       return 'Use your authenticator app';
+    case 'backup_code':
```

Comment:
🛑 These should be returning localization keys, not strings. I'll handle this right away with a new commit.

Reply:
I can do it as well in a separate PR if you would like

Reply:
Posting here for extra visibility, I handled the missing keys with a separate commit. Thanks Hari!